sdk/core/loader/types.h

```diff
@@ -523,6 +534,11 @@ namespace loader
 				return compartments[Allocator];
 			}
 
+			[[nodiscard]] const PrivilegedCompartment &token_server() const
```

Comment:
Maybe add a comment here explaining what it is?  I don't really like the token_server name anymore: it's not a token server, it's a helper library that does a fast path for some allocator functions.  We possibly want to move `heap_can_free` and `heap_quota_remaining` it as well at some point, since these are pure functions that depend only on access to the unseal capability.

Reply:
How about `ocap_fast_paths`?

Reply:
I don't think we use ocap anywhere (also, see the coding style guides about not using abbreviations).  `token_fast_paths` would work for its current use, `allocator_fast_paths` seems better since it is explicitly fast paths for functionality provided by the allocator compartment.

Reply:
I didn't want to use `allocator_fast_paths` because _static_ software-defined capabilities aren't really tied to the allocator, despite the current implementation.

Reply:
I think those should go in a separate `scheduler_fast_paths` privileged library.

Reply:
Oh, and I eventually want to rewrite the `event_*` functions on top of futex and move them out of the scheduler, so most of those won't actually need a fast path.  The scheduler will grow  `FUTEX_WAIT_BITSET` and `FUTEX_WAKE_BITSET` analogues and then the event channel things can be a separate (optional) library.